### PR TITLE
Regen install bundle for 2.0.x

### PR DESCRIPTION
**Pull request by dmvolod**

### CODE REVIEW

File: pkg/apis/infinispan/v1/infinispan_types.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ const (
 	// Additional caches can be created, but only as copies of default cache.
 	ServiceTypeCache ServiceType = "Cache"
 
-	// Deploys Infinispan to act like a data grid.
+	// Deploys Infinispan to act like a Data Grid.
```
Comment (rigazilla) [inline pkg/apis/infinispan/v1/infinispan_types.go:41]: do we need it uppercase?
```suggestion
	// Deploys Infinispan to act like a data grid.
```

Comment (dmvolod) [reply pkg/apis/infinispan/v1/infinispan_types.go:41]: Yes, sure, it more type than name in this case

**Comment by dmvolod** [comment]
@rigazilla looks like we should also regen OLM code. Is it automated or not right now?
I didn't see any CRD update in the OLM folder form the `deploy` CRD's.

**Comment by rigazilla** [comment]
> @rigazilla looks like we should also regen OLM code. Is it automated or not right now?
> I didn't see any CRD update in the OLM folder form the `deploy` CRD's.

I'm updating it manually atm.
There was a partial automation process in place, but it's has become obsolete very soon.
We need something new with support for the new bundle format

**Comment by dmvolod** [comment]
merged, thanks, @rigazilla

**Pull request merged**